engine/shared/library/sharedRandom/src/shared/FastRandomGenerator.h

```diff
@@ -16,18 +16,18 @@ class FastRandomGenerator
 {
 public:
```

Comment:
this whole thing can probably be replaced with a std native lib since its not 1999 STLPort anymore